components/_patterns/01-atoms/headings/01-heading~title-xl.yml

```diff
@@ -0,0 +1,4 @@
+heading_level:
+  "1"
+heading_content:
+  "Ithaca College Style Guide"
```

Comment:
This should probably be changed to say Emulsify Style Guide along with the other heading levels

Reply:
Ditto.